osu.Game.Rulesets.Catch.Tests/Mods/TestSceneCatchModRelax.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Framework.Testing;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Catch.Mods;
+using osu.Game.Rulesets.Catch.Objects;
+using osu.Game.Rulesets.Catch.UI;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Objects.Types;
+using osu.Game.Tests.Visual;
+using osuTK;
+
+namespace osu.Game.Rulesets.Catch.Tests.Mods
+{
+    public class TestSceneCatchModRelax : ModTestScene
+    {
+        protected override Ruleset CreatePlayerRuleset() => new CatchRuleset();
+
+        [Test]
+        public void TestModRelax() => CreateModTest(new ModTestData
+        {
+            Mod = new CatchModRelax(),
+            Autoplay = false,
+            PassCondition = passCondition,
+            Beatmap = new Beatmap
+            {
+                HitObjects = new List<HitObject>
+                {
+                    new Fruit
+                    {
+                        X = CatchPlayfield.CENTER_X,
+                        StartTime = 0
+                    },
+                    new Fruit
+                    {
+                        X = 0,
+                        StartTime = 250
+                    },
+                    new Fruit
+                    {
+                        X = CatchPlayfield.WIDTH,
+                        StartTime = 500
+                    },
+                    new JuiceStream
+                    {
+                        X = CatchPlayfield.CENTER_X,
+                        StartTime = 750,
```

Comment:
Due to the closeness between each fruit's start time, running the test with 2x the rate will fail, distancing them out will fix it:
```suggestion
                        StartTime = 0
                    },
                    new Fruit
                    {
                        X = 0,
                        StartTime = 500
                    },
                    new Fruit
                    {
                        X = CatchPlayfield.WIDTH,
                        StartTime = 1000
                    },
                    new JuiceStream
                    {
                        X = CatchPlayfield.CENTER_X,
                        StartTime = 1500,
```

Reply:
not sure how relevant that is

Reply:
Better mentioning about it rather than staying silent, though forgot to mention, my suggestion will only hide it, it can be replicated with quadruple the speed, haven't gone through what exactly is going on yet.

Reply:
I think it's fine for now.